exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -112,18 +119,22 @@ class OTLPExporterMixin(ABC):
 
     def __init__(
         self,
-        endpoint: str = "localhost:55680",
+        endpoint: str,
+        insecure: bool,
         credentials: ChannelCredentials = None,
         metadata: tuple = None,
     ):
         super().__init__()
 
+        endpoint = endpoint or os.environ.get("OTEL_EXPORTER_OTLP_ENDPOINT") or "localhost:55680"
+        insecure = insecure or os.environ.get("OTEL_EXPORTER_OTLP_INSECURE") or True # TODO(jan25): verify spec for default value. This affects how credentials are passed
```

Comment:
Does this mean values like `"false"`, `"no"`, `"off"` and their upper-case versions will evaluate to true?

Reply:
I think we should use `opentelemetry.configuration.Configuration` class to read these env vars. I think automatically casts some values to Python types and is future proof in the sense that any changes to how we prefix env vars or whether we support config files in future will not warrant a change here.

Reply:
Thanks for the hint. I adjusted the implementation to use `Configuration` object.